e2e/kots-release-install-v3/nginx-app-helm-v1beta2.yaml

```diff
@@ -32,6 +32,14 @@ spec:
         repository: repl{{ HasLocalRegistry | ternary LocalRegistryNamespace "anonymous/registry.replicated.com/library" }}/replicated-sdk-image
       imagePullSecrets:
         - name: '{{repl ImagePullSecretName }}'
+    ConfigItemsConfigMapData:
+      text: repl{{ ConfigOption "text" }}
+      password: repl{{ ConfigOption "password" }}
```

Comment:
early days so not the biggest deal, but you wouldn't usually put passwords in a ConfigMap, but instead a secret.